template/osfamilymap.yaml

```diff
@@ -12,6 +15,17 @@ RedHat:
 Suse:
   pkg: template-suse
 
+Gentoo: {}
+
 Arch:
   pkg: template-arch
-  
+
```

Comment:
Perhaps change one of these to
```yaml
Arch:
  service:
    name: template-arch
```
as an example that different vars can be added for different `os_family`s?

Reply:
Yes, definitely.  I was leaving this for a future PR, as we (hopefully) adapt the formula to actually be functional.  But another example right now is good.